src/agent.test.ts

```diff
@@ -22,9 +23,9 @@ describe('Agent', () => {
     }
   })
 
-  it('watches screen frame before calling `get()`', async () => {
+  it('loads sources when created', async () => {
+    // Checking by checking whether the screen frame is watched
```

Comment:
```suggestion
    // Checking whether the screen frame is watched
```

Reply:
The full sentence is "Checking whether agent loads entropy sources when created by checking whether the screen frame is watched". How would you write it briefly?

Reply:
oh, got it, ty